src/services/accomodation.services.js

```diff
@@ -11,6 +11,36 @@ class accomodation {
   static async findAccomodation(accId) {
     return Queries.findByOneAttribute(db.accomodation, { id: accId });
   }
+
+  /**
+     * create an accomodation
+     * @param { Object } newAccomodation accomodation id
+     * @returns {array} data the data to be returned.
+     */
+  static async createAccomodation(newAccomodation) {
+    const condition = { name: newAccomodation.name, locationId: newAccomodation.locationId };
+    return Queries.findOrCreate(db.accomodation, newAccomodation, condition);
+  }
+
+  /**
+     * upload an image
+     * @param { Object } newImage contains imageUrl and recordId
+     * @returns {array} data the data to be returned.
+     */
+  static async uploadImage(newImage) {
```

Comment:
@Jajabenit250 This function seems to be saving an image to a table not uploading it.